ts/lib/socket.ts

```diff
@@ -92,7 +107,7 @@ export class ConstellationSocket extends EventEmitter {
 
         return new ConstellationSocket.Promise((resolve, reject) => {
             let replyListener;
-            let timeout = setTimeout(() => {
+            let timeout: NodeJS.Timer = setTimeout(() => {
```

Comment:
Rm type, this is designed to be used on both node and in browser and in the browser this returns a number